api/auth/index.test.js

```diff
@@ -143,7 +143,7 @@ tap.test('authentication setup', async authTest => {
       im: 'weasel',
       ir: 'baboon'
     };
-    serializeUser.yields(null, 'unique-session-id');
+    serializeUser.resolves('unique-session-id');
```

Comment:
Curious if you think we should add a tech debt issue to add a test for the case where `serializeUser` throws an error?